config/notifiers.go

```diff
@@ -579,3 +589,46 @@ func (c *PushoverConfig) UnmarshalYAML(unmarshal func(interface{}) error) error
 	}
 	return nil
 }
+
+// TODO: Move to common?
```

Comment:
What is this comment about? `SigV4Config` only? If so, please move the TODO right before its `type` definition.